modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -2495,10 +2506,11 @@ def _callable_func(self, func, axis, *args, **kwargs):
         if callable(func):
             func = wrap_udf_function(func)
 
-        new_modin_frame = self._modin_frame.apply_full_axis(
-            axis, lambda df: df.apply(func, axis=axis, *args, **kwargs)
+        return self.__constructor__(
+            self._modin_frame.apply_full_axis(
+                axis, lambda df: df.apply(func, axis=axis, *args, **kwargs)
+            )
         )
-        return self.__constructor__(new_modin_frame)
```

Comment:
this change seems unrelated to the PR itself, it's some small code refactoring, we should not be squeezing unrelated things in PRs